.vib/supabase/cypress/cypress/integration/supabase_spec.js

```diff
@@ -0,0 +1,143 @@
+/// <reference types="cypress" />
+import {
+  random
+} from '../support/utils';
+
+/*
+  Out of the possible components, these are the ones that can be tested without
+  having to create a full Supabase development project. Here we check
+
+     - The Authorization service: checking that we can create a user
+     - The Postgres Meta service: checking that we can interact with the database
+     - The Storage service: checking that we can create a bucket
+*/
+
+it('can create a user and generate a token to it (auth)', () => {
+  // Source: https://github.com/netlify/gotrue
+  cy.fixture('users').then((users) => {
+    const userPayload = {
+      email: `${random}${users.newUser.email}`,
+      password: `${users.newUser.password}-${random}`,
+    };
+    cy.request({
+      method: 'POST',
+      headers: { apikey: Cypress.env('serviceKey') },
+      url: '/auth/v1/signup',
+      body: userPayload,
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      expect(JSON.stringify(response.body)).to.have.string(userPayload.email);
+    });
+
+    cy.request({
+      method: 'POST',
+      headers: { apikey: Cypress.env('serviceKey') },
+      // This one works with URL parameters, not with a body
+      url: `/auth/v1/token?grant_type=password&username=${userPayload.email}&password=${userPayload.password}`,
```

Comment:
When there are several query params we usually prefer to use the `qs` option in the method, which improves readability. For instance:

https://github.com/bitnami/charts/blob/913d72139b8e743fdc55caf273198e9465282d4b/.vib/solr/cypress/cypress/integration/solr_spec.js#L47-L53